### ISSUE TITLE
Deep learning: learning rate search

### ISSUE THREAD

**Issue by choshin84** [choshin84/learning_memo#34]
# Tweet summary
Try training with range of learning rate with exponential steps in each cycle, i.e. 3-5 epochs, then learn from learning curve

# Useful link
https://www.pyimagesearch.com/2019/08/05/keras-learning-rate-finder
